### PR TITLE
Feat/add on blur callback

### PR DESCRIPTION
**Pull request by arnau-rius**
- add onBlur callback to `molecule/autosuggest` and `atom/textArea`

### CODE REVIEW

File: components/atom/textarea/src/index.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const DEFAULT_PROPS = {
 
 const AtomTextarea = ({
   onChange = DEFAULT_PROPS.onChange,
+  onBlur = DEFAULT_PROPS.onBlur,
```
Comment (oriolpuig) [inline components/atom/textarea/src/index.js:25]: This onBlur is not defined at DEFAULT_PROPS constant, isn't it?

File: components/atom/textarea/src/index.js [hunk 2]
```diff
@@ -14,14 +14,10 @@ const TEXTAREA_STATES = {
   ALERT: 'alert'
 }
 
-const DEFAULT_PROPS = {
-  size: SIZES.SHORT,
-  onChange: () => {}
-}
-
 const AtomTextarea = ({
-  onChange = DEFAULT_PROPS.onChange,
```
Comment (arnau-rius) [inline components/atom/textarea/src/index.js:23]: for me it is better to define the defaultProps as default arguments (JS native way) but I don't know why this is the only component in sui world that defines the default props in such way. Ideas?

Comment (oriolpuig) [reply components/atom/textarea/src/index.js:23]: I prefer this one instead of defaultProps.

**Review by oriolpuig — APPROVED** [review]

File: components/atom/textarea/src/index.js (hunk 2 above)
Comment (davecarter) [reply components/atom/textarea/src/index.js:23]: Using default props makes components API more resilient and is kinda more declarative...
So, what else? 😄

**Pull request merged**